packages/locale/lang/de.ts

```diff
@@ -2,7 +2,7 @@ export default {
   name: 'de',
   el: {
     breadcrumb: {
-      label: 'Breadcrumb', // to be translated
+      label: 'Brotkrümel',
```

Comment:
`Brotkrümel` is the literal translation of `Breadcrumb`, and it seems to be less common in tech articles since they usually use the English name `Breadcrumb` directly without translating, so I am not very sure if I should translate it here.